### ISSUE TITLE
Case issue on postgres table names

### ISSUE THREAD

**Issue by ian-at-cos** [datacontract/datacontract-cli#310]
I have the following in my data contract:
```
models:
  PaymentLinksV1:
    type: table
    fields:
    ...
```
However when I try to test the contract, I receive this error:
<img width="1011" alt="Screenshot 2024-07-03 at 13 37 42" src="https://github.com/datacontract/datacontract-cli/assets/159006970/58a6c6fe-0daa-4871-84eb-ed895109b243">

It seems the YAML is correctly parsed, and case of the table name (PaymentLinksV1) is correctly retained - but then in the error message (relation does not exist) the table name is shown lowercase.

It looks like the table is not found because at some point the tablename is converted to lowercase. I suggest this because if I try this same test with a postgres table that has a lowercase-only name, I don't have this problem.

I would appreciate advice on this, if it's an issue or something I've missed. If the answer is just to only use lowercase table names then that's an option too.

**Comment by jochenchrist** [comment]
Yes, I confirm this is a bug. Working on it.

**Comment by jochenchrist** [comment]
Thanks for reporting.
Fix will come with `v0.10.9`.

**Comment by ian-at-cos** [comment]
Thanks for publishing the update, I have tried again with `v0.10.9`. The test now runs, but there are still some issues.

This is the error message I get (repeated twice for each run):

```
Testing datacontract.yaml
ERROR:soda.scan:[17:28:08] Query execution error in postgres.PaymentLinksV1.aggregation[0]: relation "analytics.paymentlinksv1" does not exist
LINE 13: FROM analytics.PaymentLinksV1
              ^

SELECT 
  COUNT(CASE WHEN "uuid" IS NULL THEN 1 END),
  COUNT(CASE WHEN "accountUuid" IS NULL THEN 1 END),
  COUNT(CASE WHEN "correlationId" IS NULL THEN 1 END),
  COUNT(CASE WHEN "language" IS NULL THEN 1 END),
  COUNT(CASE WHEN NOT (language IS NULL) AND NOT (LENGTH(language) <= 1) THEN 1 END),
  COUNT(CASE WHEN "currency" IS NULL THEN 1 END),
  COUNT(CASE WHEN NOT (currency IS NULL) AND NOT (LENGTH(currency) <= 3) THEN 1 END),
  COUNT(CASE WHEN "amount" IS NULL THEN 1 END),
  COUNT(CASE WHEN "status" IS NULL THEN 1 END),
  COUNT(CASE WHEN "createdAt" IS NULL THEN 1 END),
  COUNT(CASE WHEN "deletedAt" IS NULL THEN 1 END) 
FROM analytics.PaymentLinksV1
ERROR:soda.scan:  | relation "analytics.paymentlinksv1" does not exist
  | LINE 13: FROM analytics.PaymentLinksV1
  |               ^

ERROR:soda.scan:[17:28:08] Query execution error in postgres.PaymentLinksV1."uuid".duplicate_count: relation "analytics.paymentlinksv1" does not exist
LINE 4:     FROM analytics.PaymentLinksV1
                 ^


WITH frequencies AS (
    SELECT COUNT(*) AS frequency
    FROM analytics.PaymentLinksV1
    WHERE "uuid" IS NOT NULL
    GROUP BY "uuid")
SELECT COUNT(*)
FROM frequencies
WHERE frequency > 1
ERROR:soda.scan:  | relation "analytics.paymentlinksv1" does not exist
  | LINE 4:     FROM analytics.PaymentLinksV1
  |                  ^

ERROR:soda.scan:[17:28:08] Metrics 'missing_count' were not computed for check 'missing_count("uuid") = 0'
ERROR:soda.scan:[17:28:08] Metrics 'duplicate_count' were not computed for check 'duplicate_count("uuid") = 0'
ERROR:soda.scan:[17:28:08] Metrics 'missing_count' were not computed for check 'missing_count("accountUuid") = 0'
ERROR:soda.scan:[17:28:08] Metrics 'missing_count' were not computed for check 'missing_count("correlationId") = 0'
ERROR:soda.scan:[17:28:08] Metrics 'missing_count' were not computed for check 'missing_count("language") = 0'
ERROR:soda.scan:[17:28:08] Metrics 'invalid_count' were not computed for check 'invalid_count(language) = 0'
ERROR:soda.scan:[17:28:08] Metrics 'missing_count' were not computed for check 'missing_count("currency") = 0'
ERROR:soda.scan:[17:28:08] Metrics 'invalid_count' were not computed for check 'invalid_count(currency) = 0'
ERROR:soda.scan:[17:28:08] Metrics 'missing_count' were not computed for check 'missing_count("amount") = 0'
ERROR:soda.scan:[17:28:08] Metrics 'missing_count' were not computed for check 'missing_count("status") = 0'
ERROR:soda.scan:[17:28:08] Metrics 'missing_count' were not computed for check 'missing_count("createdAt") = 0'
ERROR:soda.scan:[17:28:08] Metrics 'missing_count' were not computed for check 'missing_count("deletedAt") = 0'
```
Here's what the result looks like:
<img width="927" alt="Screenshot 2024-07-03 at 17 29 47" src="https://github.com/datacontract/datacontract-cli/assets/159006970/9487338d-08ef-4862-a2b6-fb630460de54">

Interestingly in the results there's no mention of this 'maxLength' rule, despite the fact that it should fail for all current records in the table:
```
language:
  type: varchar
  required: true
  maxLength: 1
  example: "de"
```
...even though in the SQL dumped in the error message there's mention of it:
```
COUNT(CASE WHEN NOT (language IS NULL) AND NOT (LENGTH(language) <= 1) THEN 1 END),
```
